core/tx_pool.go

```diff
@@ -463,6 +463,11 @@ func (pool *TxPool) reset(oldHead, newHead *types.Header) {
 	pool.promoteExecutables(nil)
 }
 
+// GetTxPoolSize returns tx pool size.
+func (pool *TxPool) GetTxPoolSize() uint64 {
```

Comment:
is this class actually used? I thought this was not being used yet until txPool integration happens.

Reply:
It is used for tx pool size metric.

Reply:
but we are not using TxPool yet. cc: @rlan35 @mikedoan ?